modules/codemod/lib/v5/spec/renameCoreImports.spec.ts

```diff
@@ -0,0 +1,27 @@
+import {expectTransformFactory} from './expectTransformFactory';
+import transformer from '../renameCoreImports';
+const context = describe;
+
+const expectTransform = expectTransformFactory(transformer);
+
+describe('Canvas Kit Core Imports Codemod', () => {
+  context('when importing from canvas-kit-react/core', () => {
+    it('should do properly transform core to tokens', () => {
+      const input = `import {colors} from "@workday/canvas-kit-react/core";`;
+      const expected = `import {colors} from "@workday/canvas-kit-react/tokens";`;
+
+      expectTransform(input, expected);
+    });
+  });
+
+  context('when importing from canvas-kit-labs-react/core', () => {
+    // import type from '@workday/canvas-kit-labs-react-core
+    // import type, { type, colors, StaticStates } '@workday/canvas-kit-labs-react/core';
```

Comment:
What is the purpose of these comments?

Reply:
Oh, sorry. This is still a draft. Those are potential test cases.